### PR TITLE
[TACHYON-771] Standardize FileUtils functionality

### PR DESCRIPTION
**Pull request by fanshiqing**
https://tachyon.atlassian.net/browse/TACHYON-771
Through reviewing the codebase, it appears that 'path' appears more frequently than 'Files' as input parameters of 'FileUtils' 's util methods. Hence we will take the former for standardization.

### CODE REVIEW

**Comment by apc999** [comment]
+1 LGTM! Thanks for this patch --- actually I was about to create a JIRA exactly on this :)

File: common/src/main/java/tachyon/util/io/FileUtils.java [hunk 1]
```diff
@@ -145,40 +152,42 @@ public static void createBlockPath(String path) throws IOException {
...
-      throw new IOException("Failed to delete " + file);
+      throw new IOException("Failed to delete " + path);
     }
   }
 
   /**
    * Creates a file and its intermediate directories if necessary.
    *
-   * @param file the file to create
+   * @param pathPath pathname string of the file to create
```
Comment (gpang) [inline common/src/main/java/tachyon/util/io/FileUtils.java]: the param name is "filePath"

File: common/src/main/java/tachyon/util/io/FileUtils.java [hunk 2]
```diff
@@ -30,6 +30,11 @@
 import tachyon.TachyonURI;
 import tachyon.thrift.InvalidPathException;
 
+/**
+ * Provides utility methods for working with files and directories.
+ * 
+ * Incoming methods please better take pathname string as parameters for standardization.
```
Comment (gpang) [inline common/src/main/java/tachyon/util/io/FileUtils.java]: Please reword this to something like: "By convention, methods take file path strings as parameters."

**Comment by gpang** [comment]
Thanks for the PR! I just had a few minor comments.

**Comment by fanshiqing** [comment]
@gpang  @apc999 Thanks for your review! I have fix them :)

**Comment by gpang** [comment]
LGTM

**Comment by apc999** [comment]
+1 LGTM
